### ISSUE TITLE
Reference the Repo Id wiki in the project.md template wiki

### ISSUE THREAD

**Issue by jbubar** [hackforla/website#1478]
### Dependency

- #1534

### Overview
Sometimes when people are building know project.md files they need to find the github repo Id. We should reference it in the wiki.

### Action Items
- [x] link to the repo ID article in the [template for a project.md file wiki article](https://github.com/hackforla/website/wiki/project.md-file-template) using the following text.

`If you cannot find the Github Repository ID refer to this [wiki article](https://github.com/hackforla/website/wiki/How-to-get-repository-ID-for-a-new-project-card)`

- [x] take a screen shot of that change and post it in this issue

Screenshot before change:
![hfla-1478-b4](https://user-images.githubusercontent.com/97845061/227069412-da3d487d-c9af-482a-984c-93acfe195e8f.JPG)

Screenshot after change:
![hfla-1478-after](https://user-images.githubusercontent.com/97845061/227070036-868a7292-ef7e-4172-aa1e-7930735f927a.JPG)

- [x] put this in the links/questions/review column of the project board so that it can be reviewed
- [ ] close this issue upon completion

### Resources/Instructions
[template for a project.md file wiki article](https://github.com/hackforla/website/wiki/project.md-file-template)
[how to find the repo ID wiki article](https://github.com/hackforla/website/wiki/How-to-get-repository-ID-for-a-new-project-card)

**Comment by akibrhast** [comment]
@jbubar What are the labels?Projects? and so on in regards to this issue?

**Comment by janice87** [comment]
Availability: Wed-Fri 5-9PM EST
ETA: 3/23/23

**Comment by janice87** [comment]
This is my first time editing a markdown file. I cloned the wiki locally and saved the changes in the markdown file. I tried pushing the changes, but it's saying there are no changes. When I go to the pull request, it says there isn't anything to compare.

![image](https://user-images.githubusercontent.com/97845061/226784080-d339d2ae-179a-4d98-bc2c-ec2b8ef6169e.png)

**Comment by ExperimentsInHonesty** [comment]
Hi @janice87 

I recommend posting in the hfla-site slack channel with the link to your comment https://github.com/hackforla/website/issues/1478#issuecomment-1478828173 so that your peers can help you.  If that works, move this back to the prioritized backlog and note what you ended up doing, in a comment on the issue.  

If you don't end up getting help in the slack channel, then please go to office hours on Thursday.

**Comment by janice87** [comment]
Thank you, I posted this in the hfla site slack channel and will see if any one can help. Otherwise I will attend the office hours to resolve this issue.

**Comment by janice87** [comment]
I posted the question in slack and found out that editing the wiki can be made directly on GitHub. 

Screenshot before change:
![1](https://user-images.githubusercontent.com/97845061/227384780-42b99df3-49e1-4d42-bc99-d3daccbd12b2.JPG)

Screenshot after change: 
![2](https://user-images.githubusercontent.com/97845061/227384836-88b04b9c-8319-4c6e-8418-8d16c44bba6c.JPG)

I added the link that references the repo ID wiki and the update can be seen [here](https://github.com/hackforla/website/wiki/project.md-file-template). Once this  is reviewed and approved, I will mark the last item off and close this issue.

**Comment by ExperimentsInHonesty** [comment]
@janice87 I have asked the merge team to review your changes and sign-off or give feedback.

**Comment by jdingeman** [comment]
Hi @janice87 - the link leads to a new article page. I think the link in this issue may be outdated and doesn't point to the correct source anymore. 
![image](https://user-images.githubusercontent.com/88953806/228077711-f9321153-080b-445f-9f1c-da0f9e40dc67.png)

**Comment by janice87** [comment]
Hi @jdingeman I searched other wikis to see if there are any other pages related to finding a repo ID, but didn't see any. I am wondering if the link to the wiki article is helpful, since the paragraph above where we provided the link mentions "To find the id, go to the GitHub homepage for the project's main repository and search the page source for <meta name="octolytics-dimension-repository_id" content="#####">; the number in the 'content' field is the id you want.". I feel like this addresses what the outdated link should have, thoughts?
